src/util/mapbox.js

```diff
@@ -17,6 +17,10 @@ function makeAPIURL(urlObject: UrlObject, accessToken: string | null | void): st
     urlObject.protocol = apiUrlObject.protocol;
     urlObject.authority = apiUrlObject.authority;
 
+    if (apiUrlObject.path !== '/') {
+        urlObject.authority = `${urlObject.authority}${apiUrlObject.path}`;
```

Comment:
Could this prepend `apiUrlObject.path` to `urlObject.path`, rather than appending it to `authority`? That would be clearer and more robust.